libraries/ms-common/test/test_gpio.c

```diff
@@ -2,13 +2,18 @@
 
 #include "extra_unity.h"
 #include "gpio.h"
+#include "gpio_cfg.h"
 #include "unity.h"
 
 void setup_test(void) { }
 
 void teardown_test(void) { }
 
-#define INVALID_PORT_PIN 100
+#define VALID_PIN 0
+#define VALID_PORT 0
```

Comment:
I just incremented to 1. This might be an issue long term for APIs as, some valid port pin combinations may get invalidated between boards we'll need to figure that out when the time comes to it.